BlackNectar/BlackNectarTests/DistanceCalculationTests.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  DistanceCalculationTests.swift
+//  BlackNectar
+//
+//  Created by Cordero Hernandez on 1/16/17.
+//  Copyright © 2017 Black Whole. All rights reserved.
+//
+
+@testable import BlackNectar
+
+import CoreLocation
+import XCTest
+
+class DistanceCalculationTests: XCTestCase {
+    
+    func testGetDistance() {
+        
+        let usersLatitude: CLLocationDegrees = 40.6782
+        let usersLongitude: CLLocationDegrees = -73.9442
+        let userslocation = CLLocationCoordinate2D(latitude: usersLatitude, longitude: usersLongitude)
+        
+        let restaurantsLatitude: CLLocationDegrees = 34.14341
+        let restaurantsLongitude: CLLocationDegrees = -118.392
+        let restaurantsLocation = CLLocationCoordinate2D(latitude: restaurantsLatitude, longitude: restaurantsLongitude)
+        
+        XCTAssertNotNil(DistanceCalculation.getDistance(userLocation: userslocation, storeLocation: restaurantsLocation))
```

Comment:
+ this is taking up more time than I thought 
+ this shouldn't take me this long but for some reason it is 
+ working on it